src/components/filter/MessagesFilterPanel.tsx

```diff
@@ -64,25 +78,35 @@ const MessagesFilterPanel = () => {
 	}, [filterStore.isSoftFilter]);
 
 	React.useEffect(() => {
-		setSSEFilter(messagesStore.filterStore.sseMessagesFilter);
 		setCurrentValues({
 			type: '',
 			body: '',
 			attachedEventIds: '',
 		});
-	}, [messagesStore.filterStore.sseMessagesFilter]);
+	}, []);
 
 	const submitChanges = React.useCallback(() => {
+		if (filter) {
+			const timestamp = Date.now();
+			if (Object.values(filter).some(v => v.values.length > 0)) {
```

Comment:
same as in eventFilterPanel, move this logic to store and use it messagesStore.applyFilter